src/wkwebview/mod.rs

```diff
@@ -639,7 +639,7 @@ r#"Object.defineProperty(window, 'ipc', {
       let store = config.websiteDataStore();
       let all_data_types = WKWebsiteDataStore::allWebsiteDataTypes();
       let date = NSDate::dateWithTimeIntervalSince1970(0.0);
-      let handler = block2::RcBlock::new(|| {});
+      let handler = block2::RcBlock::new(|| {}).copy();
```

Comment:
Hmm, this copy should not be necessary in `block2`?

Reply:
You're right! `RcBlock` is already in the heap. Thanks!